[Docathon][Fix Doc Format No.5、6] Fix example code format in `paddle.linalg.lu_solve` and `paddle.tensor_split`

wanglezz opened the pull request:
<!-- TemplateReference: https://github.com/PaddlePaddle/Paddle/wiki/PULL-REQUEST-TEMPLATE--REFERENCE -->
<!-- Demo: https://github.com/PaddlePaddle/Paddle/pull/24810 -->

### PR Category
<!-- One of [ User Experience | Execute Infrastructure | Operator Mechanism | CINN | Custom Device | Performance Optimization | Distributed Strategy | Parameter Server | Communication Library | Auto Parallel | Inference | Environment Adaptation ] -->
User Experience

### PR Types
<!-- One of [ New features | Bug fixes | Improvements | Performance | BC Breaking | Deprecations | Docs | Devs | Not User Facing | Security | Others ] -->
Docs

### Description
<!-- Describe what you’ve done -->
1. 修复了`tensor-split-example-5`代码块拼写错误
2. 给`lu_solve`函数添加了`.. code-block:: python`并缩进


issue链接 ：PaddlePaddle/docs#7435
@Echo-Nie @sunzhongkai588

wanglezz committed:
fix doc tensor_split_cn lu_solve_cn

wanglezz committed:
Merge branch 'develop' of https://github.com/PaddlePaddle/Paddle into docsfix

cszdrg committed:
correlation supports big tensor (#75383)

* fix

* fix test

* fix

fxyfxy777 committed:
[compat]add set/get rng_state get_device_module (#75435)

* add set/get rng_state get_device_module

zty-king committed:
Add the test about the sharded_state_dict of optimizer  (#75067)

* fix the share_weight_bug

* add note

* add the unit test

* set the timeout

* add more test

* Trigger CI rebuild

* fix the CmakeLists

HydrogenSulfate committed:
Fix get mem info (#75381)

* fix int arg for get_mem_info

* fix UT

* fix for UT cov

* fix UT

bigwhite37 committed:
update XPU ReLU grad callers to the new API signature (#75321)

SigureMo commented on python/paddle/tensor/linalg.py:
```diff
@@ -3428,19 +3428,21 @@ def lu_solve(
+            >>> A = paddle.to_tensor([[3, 1], [1, 2]], dtype="float64")
+            >>> b = paddle.to_tensor([[9, 8], [9, 8]], dtype="float64")
+            >>> lu, p = paddle.linalg.lu(A)
+            >>> x = paddle.lu_solve(b, lu, p)
```
```suggestion
            >>> x = paddle.linalg.lu_solve(b, lu, p)
```

SigureMo reviewed (commented):
自己看 static-check 报错改一下，无论是示例代码检查还是类型提示检查这里都挂了

wanglezz committed:
Update python/paddle/tensor/linalg.py

Co-authored-by: Nyakku Shigure <sigure.qaq@gmail.com>

co63oc committed:
rename analyzer_detect_functional_mkldnn_tester_deprecated (#75464)

co63oc committed:
rename test/cpp/fluid/mkldnn/ (#75465)

co63oc committed:
use op_test.get_places in test_logical_op (#75470)

co63oc committed:
use paddle.set_device(place) in test_dropout_op (#75471)

zhengshengning committed:
paddle.tanh Grad and torch alignment (float16) (#75454)

co63oc committed:
remove check cuda >= 10.0 (#75263)

ApricityXX committed:
2nd-batch-24-修复函数命名和预期语义 (#75439)

ApricityXX committed:
2nd-batch-42to43-代码执行流逻辑错误 (#75453)

scyyh11 committed:
feat: Add ComplexType support for Softplus and CudaSoftplus functors (#75484)

- Implemented SoftplusFunctor and CudaSoftplusFunctor for ComplexType<T>.
- Added attributes beta and threshold to both functors.
- Enhanced operator() methods to handle complex number calculations for softplus activation.

co63oc committed:
add include nvtx3/nvToolxExt.h (#75479)

qjyyy77 committed:
Update README_ja.md (#75234)

update numbers of developers and companies.

qjyyy77 committed:
Update README_cn.md (#75233)

update numbers of developers and companies.

qjyyy77 committed:
Update README.md (#75232)

update numbers of developers and companies.

ApricityXX committed:
2nd_batch_31 (#75442)

ApricityXX committed:
2nd-batch-35-修复代码语法错误 (#75448)

* 2nd_batch_35

* 923

* 923

ApricityXX committed:
修复fix_any.h在cpplint上的报错 (#75438)

* fix_any.h

* test2

* test3

* test4

* test5

* test6

Dayuxiaoshui committed:
【UnitTestFix No.9】: Fix test_memcpy_op unittest error and support PIR API compatibility (#75395)

* fix: Fix test_memcpy_op unittest error and support PIR API compatibility

PR Category
Operator Mechanism

PR Types
Bug fixes

Description
Fix test_memcpy_op unittest error and support PIR API compatibility.
The Paddle framework has migrated from old IR to new PIR (Program IR), causing API compatibility issues in static graph tests.
test_memcpy_op unit test now supports PIR API and uses dynamic graph mode for better compatibility.
Fixed data type checking issues where paddle.float32 corresponds to different objects in different contexts.

* refactor: Replace Chinese comments with English comments in test_memcpy_op.py

- Replace all Chinese comments with English equivalents
- Improve code readability for international developers
- Maintain consistent comment language throughout the codebase

* Fix test_memcpy_op unittest compatibility with PIR API

- Replace static graph API with dynamic graph mode to avoid PIR compatibility issues
- Fix create_var and append_op calls that are not available in PIR API
- Add CUDA fallback handling for CPU-only environments
- Ensure all tests pass in both CUDA and CPU environments
- Maintain original test logic and assertions

PR Category: Operator Mechanism
PR Types: Bug fixes
Description: Fix memcpy unittest error and support PIR API compatibility. The test now uses dynamic graph mode to avoid PIR API compatibility issues while maintaining the same test coverage.

* Fix test_memcpy_op.py for GPU environment compatibility

- Remove dependency on op_test module to fix import errors
- Add local get_device_place function implementation
- Fix static graph mode issues in API test
- Ensure all tests pass in GPU environment with CUDA support

All 5 tests now pass successfully:
- test_gpu_copy_to_pinned: GPU to GPU Pinned memory copy
- test_pinned_copy_gpu: GPU Pinned memory to GPU copy
- test_hip_copy_bool_value: HIP environment bool value copy
- test_SELECTED_ROWS: SELECTED_ROWS type error handling
- test_api: API functionality test

* chore: fix EOF newline for test_memcpy_op.py via pre-commit end-of-file-fixer

* test(legacy): replace get_device_place with base.CUDAPlace(0); fix EOF newline

YqGe585 committed:
Add 7 api under paddle.cuda (#75342)

* add 7 api under paddle.cuda

Difers committed:
Fix dtype api en doc (#74603)

* fix docstring

* fix typo

Eddie-Wang1120 committed:
[Stride] Support Matmul Kernel with Transposed Input (#75452)

* matmul init

* refine

* refine

* refine

* refine

* refine

* refine

* refine

fxyfxy777 committed:
[Compat] Unified some cuda/device apis (#75455)

* add set/get rng_state get_device_module

co63oc committed:
Replace deprecated `paddle.dataset.uci_housing` with `paddle.text.datasets.UCIHousing` in unittests (#75487)

YqGe585 committed:
Add tf32 switch apis for Custom Device (#75462)

* add tf32 apis

* fix header

* fix cpu error

tianshuo78520a committed:
Add nigth coverage (#75348)

* Add ALL Coverage

* Add night all Coverage

zhengshengning committed:
[Precision Depth Alignment] paddle.sin and paddle.cos aligns with torch precision. (#75503)

* accuracy_stable_sin

* accuracy_stable_cos

ApricityXX committed:
2nd-batch-32to33-提取虚部浮点值错误 (#75444)

* 2nd_batch_32

* 2nd_batch_32to33

co63oc committed:
rename test/deprecated/mkldnn (#75488)

co63oc committed:
update disable_ut_xpu_kl3.local (#75490)

ApricityXX committed:
2nd-batch-04\05\06\07-去除无用死代码 (#75334)

* 2nd_batch_04

* 2nd_batch_04

ApricityXX committed:
2nd_batch_16 (#75349)

wanglezz committed:
ci: Trigger CI pipeline,test=document_fix
